web/Services/LandlordService.cs

```diff
@@ -17,6 +17,7 @@ public enum LandlordRegistrationResult
 
     public Task<LandlordRegistrationResult> RegisterLandlordWithUser(LandlordProfileModel createModel, BricksAndHeartsUser user);
     public List<PropertyDbModel> GetListOfProperties(int landlordId);
+    public Task<LandlordDbModel> GetLandlordFromId(int id);

```

Comment:
This can return null (or rather, awaiting the Task can return null), so include it in the type signature, and I'd change the method name as well
```suggestion
    public Task<LandlordDbModel?> GetLandlordIfExistsFromId(int id);
```